docs/modules/ROOT/pages/how-tos/testing/build/index.adoc

```diff
@@ -1 +1,27 @@
-= Build-time tests
\ No newline at end of file
+= Build-time tests
+
+This document covers the build-time tests (formerly known as "sanity tests") that {ProductName} runs as part of its component build pipeline. These build-time tests automatically check all application images to ensure that they're up-to-date, correctly formatted, and protected from security vulnerabilities.
+
+The component build pipeline for {ProductName} include various types of build-time tests, conducted using link:https://tekton.dev/docs/pipelines/tasks/#overview[Tekton tasks]. For validating container information, the pipeline employs https://www.conftest.dev/[Conftest]. The table below outlines the currently implemented build-time tests:
+
+.Deprecated image checks
+|===
+|Test name |Description |Failure message
+
+|image_repository_deprecated |Deprecated images are no longer maintained, leading to unresolved security vulnerabilities. | The container image must not be built from a repository  marked as 'Deprecated' in COMET
```

Comment:
COMET is likely irrelevant for external users. Can we change this?
@dirgim, should the task itself be changes as it is exposed to the community? I assume that this is only relevant for Red Hat image references.

Reply:
That's correct, the [konflux-test policy](https://github.com/konflux-ci/konflux-test/blob/9bc1111b160a215bedb4bc3cc9d3f866e9b52ff0/policies/repository/deprecated-image.rego#L7-L8) for this has already been updated to refer to the Red Hat Catalog instead of COMET.
My proposal is to use the same text here :
```
|image_repository_deprecated |Deprecated images are no longer maintained and will accumulate security vulnerabilities without releasing a fixed version. | The container image shouldn't be built from a repository that is marked as 'Deprecated' in Red Hat Catalog.
```